lib/prompts/projectNamePrompt.ts

```diff
@@ -1,12 +1,14 @@
-// @ts-nocheck
-const { promptUser } = require('./promptUtils');
-const { i18n } = require('../lang');
-const { ensureProjectExists } = require('../projects');
-const { uiAccountDescription } = require('../ui');
+import { promptUser } from './promptUtils';
+import { i18n } from '../lang';
+import { ensureProjectExists } from '../projects';
+import { uiAccountDescription } from '../ui';
 
 const i18nKey = 'lib.prompts.projectNamePrompt';
 
-const projectNamePrompt = (accountId, options = {}) => {
+export function projectNamePrompt(
```

Comment:
This is missing a return type - if you add one you _might_ be able to avoid some of the non null assertions for project name (in the open command for example)

Reply:
Added in [f82d7c8](https://github.com/HubSpot/hubspot-cli/pull/1412/commits/f82d7c89907fb62f41ed91af6d8953391bb79630) 👌